proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -171,6 +179,10 @@ service PluginService {
   // This does not include users who have implicit roles against the plugin, unless they have also been
   // assigned a role explicitly.
   rpc ListPluginContributors(ListPluginContributorsRequest) returns (ListPluginContributorsResponse);
+  // DeprecatePluginByName deprecates the plugin, if found.
+  rpc DeprecatePluginByName(DeprecatePluginByNameRequest) returns (DeprecatePluginByNameResponse);
```

Comment:
I don't think we need the `ByName` on this service, everything uses the name as a lookup key. I appreciate it's consistent with repositories, but in that service we have both ID and Name functions, which it's why it makes more sense there. What do you think?

Reply:
make sense to me 👍